mds/apis/agency_api/v0_x/service_areas.py

```diff
@@ -26,11 +27,20 @@ class AreaSerializer(serializers.ModelSerializer):
     service_area_id = serializers.UUIDField(
         source="id", help_text="Unique Area identifier (UUID)"
     )
-    service_area = MultiPolygonField(source="polygons.all")
+    start_date = utils.UnixTimestampMilliseconds(
+        source="creation_date",
```

Comment:
maybe renaming model fields would be clearer ?

Reply:
IMO `creation_date` sounds more explicit though

Reply:
I mean clearer in the sense that the field name in the APIn is the same as the column name in the DB.